package/cloudshell/iac/terraform/tagging/tag_terraform_resources.py

```diff
@@ -647,14 +649,15 @@ def start_tagging_terraform_resources(main_dir_path: str, logger, tags_dict: dic
                 LoggerHelper.write_error("Tagging terraform resources operation has FAILED !!!!!")
                 # Had to change exit(1) to "raise" so exception can be handled outside
                 # exit(1)
-                raise ValueError("Errors were found in the last validation check")
+                raise TerraformAutoTagsError("Errors were found in the last validation check")
```

Comment:
should be something like:
""Validation errors during Terraform Init/Plan when applying automated tags""